### ISSUE TITLE
What is the proper fasta file to use for 1.bwamem-link.pl?

### ISSUE THREAD

**Issue by cmcninch** [caulai/Mo17_genome_assembly#1]
When running the 1.bwamem-link.pl script, what does the "gene.fa" file on line 10 refer to? Is this a FASTA format file of the nucleotide sequences corresponding to all RNA features annotated on the assembly, based on the genome sequence? Thanks for your help!

#!/usr/bin/perl -w
#2017/4/20  

$gene_structure=$ARGV[0];  #"gene.stucture.new";
$ref_genome=$ARGV[1];      # the second genome  (Mo17/B73)
$cds_seq=$ARGV[2]:         #"gene-full-cds-double.new.fa";
$cds_loc=$ARGV[3];         #"cds.loc" 
$script_dir=$ARGV[4];      

`bwa mem -t 1  $ref_genome   gene.fa > aln.sam `;

**Comment by caulai** [comment]
Yes! 'gene.fa' is a fasta file. It was generated by the first script which was also named as 'gene-full-cds-double.new.positive.fa' file. Thanks
